digdag-standards/build.gradle

```diff
@@ -47,8 +47,6 @@ dependencies {
     compile 'com.google.code.findbugs:jsr305:3.0.1'
 
     // Newer version of jetty-client with some important bugfixes.
-    // jetty-client is used by td-client-java, and ideally we would bump the version there but
-    // td-client-java still targets JDK 7 and jetty-client 9.3.x targets JDK 8.
```

Comment:
The latest version of td-client-java uses java 8 and digdag would use td-client-java 0.8.5 by this PR. So this comment is not needed.